src/Agent.Listener/Configuration/CredentialProvider.cs

```diff
@@ -47,7 +47,7 @@ public override VssCredentials GetVssCredentials(IHostContext context)
 
             // PAT uses a basic credential
             VssBasicCredential basicCred = new VssBasicCredential("VstsAgent", token);
-            VssCredentials creds = new VssClientCredentials(basicCred);
+            VssCredentials creds = new VssCredentials(null, basicCred, CredentialPromptType.DoNotPrompt);
```

Comment:
is null the same as [this](https://github.com/Microsoft/vsts-task-lib/blob/master/powershell/VstsTaskSdk/ServerOMFunctions.ps1#L290)